src/browser/index.js

```diff
@@ -1,5 +1,5 @@
 import './vdiff.js';
 
-export { assert, aTimeout, defineCE, expect, html, nextFrame, oneDefaultPreventedEvent, oneEvent, waitUntil } from '@open-wc/testing';
+export { assert, aTimeout, defineCE, elementUpdated, expect, html, nextFrame, oneDefaultPreventedEvent, oneEvent, waitUntil } from '@open-wc/testing';
```

Comment:
I was mostly expecting folks to just use `await elem.updateComplete` in these cases... were you relying on the extra stuff they have in there? If we don't think it's needed, I can update the migration guide to just call this out.

Reply:
I can take a look at not using it and see if I have any troubles. I was just doing my best to not change the tests if I didn't have to.

Reply:
I'll take a look tomorrow

Reply:
Ok let me know. I'm OK with adding it, but yeah it always struck me as odd to abstract it away when `elem.updateComplete` was right there. And then having it fall back to `await nextFrame` if the thing _isn't_ a Lit element was just extra magic.